python_examples/basic_infection_history_simulation/simple_parameters.json

```diff
@@ -334,6 +336,29 @@
     6.7222126,
     10.24997697
 ],
+"recov_to_susc_icdf": [
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0,
+    0
```

Comment:
These don't all need to be on separate rows, if they're all zero - I think it is much less readable.

Reply:
(Ditto for other json files)